### DISCUSSION TITLE
Is there a way to cache xmake packages on a CI?

### DISCUSSION THREAD

**Discussion by yh-sb** [xmake-io/xmake#5506] [Q&A] [answered]
Is there a way to cache xmake packages on a CI using [github-action-setup-xmake](https://github.com/xmake-io/github-action-setup-xmake) or perhaps through a manual method?

Guess that a cache key can be main xmake.lua in the root of the project + build workflow .yml file (to consider xmake version change).

**Comment by waruqi** [comment, marked as answer]
https://github.com/NazaraEngine/NazaraEngine/blob/6c045e618f62cc7c1bba4a8529eadab643420175/.github/workflows/macos-build.yml#L70
